Library/Homebrew/cask/cask.rb

```diff
@@ -347,15 +342,64 @@ def to_h
       }
     end
 
-    def to_hash_with_variations
-      if loaded_from_api? && !Homebrew::EnvConfig.no_install_from_api?
-        return api_to_local_hash(Homebrew::API::Cask.all_casks[token].dup)
+    # @private
+    def to_api_hash
```

Comment:
```suggestion
    def to_internal_api_hash
```

assuming this is actually correct

Reply:
I'll just rename the formula ones too. I have no preference personally but I guess it indicates intent a bit better than before.

Reply:
Addressed in https://github.com/Homebrew/brew/pull/16798/commits/b5c9651b4624a5349202c3346c5983cd7e27ac5e